api/v1alpha1/selfnoderemediationconfig_types.go

```diff
@@ -170,6 +176,7 @@ func NewDefaultSelfNodeRemediationConfig() SelfNodeRemediationConfig {
 		Spec: SelfNodeRemediationConfigSpec{
 			WatchdogFilePath:        defaultWatchdogPath,
 			IsSoftwareRebootEnabled: defaultIsSoftwareRebootEnabled,
+			MinPeersForRemediation:  defaultMinPeersForRemediation,
```

Comment:
this is redundant, will be set by api server because of the default defined in the kubebuilder marker

Reply:
Removed the redundant line